website/content/en/docs/_index.md

```diff
@@ -1,14 +1,20 @@
 
 ---
-title: "Documentation"
-linkTitle: "Docs"
+title: "v2 Documentation"
```

Comment:
Can we think of latest as "undecorated". E.g.

Documentation <------ latest
Documentation (v0.4.0) <--- older
Documentation (v0.3.5) <---- older

Reply:
I'm worried this would get confusing if we later introduce a "pre-release" section.

docs - pre-release? dev?
docs - latest? current? 
docs - v4
docs - v3

I like the idea of the dropdown showing the current version you are browsing. https://knative.dev/docs/

Reply:
@geoffcline I like the knative look - what do you think about making it `(Pre-release)` and being slightly grey'd out as they have it?

Reply:
I'm good with the structure, we can improve on the style in the future.